javascript/packages/cli/package.json

```
@@ -50,8 +50,20 @@
     "url": "https://github.com/paritytech/zombienet/issues"
   },
   "dependencies": {
+    "@parity/zombienet-dsl-parser-wrapper": "^0.1.6",
     "@zombienet/orchestrator": "^0.0.5",
-    "@zombienet/utils": "^0.0.3"
+    "@zombienet/utils": "^0.0.3",
+    "axios": "^0.27.2",
+    "debug": "^4.3.2",
+    "nunjucks": "^3.2.3",
+    "pkg": "~5.8.0",
```

```suggestion
    "pkg": "~5.8.0",
```

pkg should be a `dev dep` since is only for packaging.